src/tokenizer.ts

```diff
@@ -196,11 +196,54 @@ export class Tokenizer {
     private isDigit(c: string): boolean {
         return /^[0-9]/.test(c);
     }
-
+	
+	private isHexa(c: string) : boolean {
```

Comment:
What IDE are you using? Could you please set it to use spaces instead of tabs for indentation please? The rest of the file uses 4 spaces for an indent.

Please fix the indentation for the rest of the file too.